### ISSUE TITLE
Linux default menu implementation in druid

### ISSUE THREAD

**Issue by 8176135** [xi-editor/druid#281]
`druid::menu::sys` does not have a default implementation for linux. 

Should it have one? I don't think there is a default menu guideline for general linux anywhere, and I also don't see it being that much different from the windows default menu.

**Comment by cmyr** [comment]
This is a good question. I'm not sure I like the `menu::sys` stuff; I've been finding it hard to use in runebender, and I end up just copy-pasting a bunch of stuff out of there and using it directly.

I don't really know what the long-term story is for menus. Mainly I would like it to be easy for people to create menus that look right on their platform. I think this is less important on linux than on windows, and less important on windows than on mac; but I haven't used linux as my daily driver in quite a few years, so it may be that standards have changed.

**Comment by LiHRaM** [comment]
Yeah, Linux suffers from the most fragmentation in these matters.
The following docs might be relevant, as we're using GTK to implement the Linux part: https://gitlab.gnome.org/GNOME/Initiatives/wikis/App-Menu-Retirement

TL;DR: Use a burger menu on the upper right side.

They're also moving towards CSD (client-side decorations, e.g. no titlebar), idk if we're doing that here, too. Definitely a different issue.
